template/src/project_name/accounts/models.py

```diff
@@ -0,0 +1,7 @@
+from django.contrib.auth.models import AbstractUser
+
+class CustomUser(AbstractUser):
+    pass
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏`pass` is not needed here